server/controllers/index.js

```diff
@@ -4,12 +4,15 @@ const { getPlaces } = require('./places')
 const { login } = require('./login')
 const { getGuides } = require('./guides')
 const { signUp } = require('./signup')
+const { isAuthenticated } = require('../middleware/auth')
 const { getArabicWords } = require('./arabicList')
 
 router.get('/places', getPlaces)
+router.get('/auth', isAuthenticated)
```

Comment:
This should be before a defined route but this needs a lot of refactoring. For now, it's okay but please write an issue on this!